tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -2763,7 +2763,7 @@ public function testCounter()
 
         $expected = 'Page 1 of 5, showing 3 records out of 13 total, starting on record 1, ';
         $expected .= 'ending on 3';
-        $result = $this->Paginator->counter($input);
+        $result = $this->Paginator->counter(['format' => $input]);
```

Comment:
This method also seems like a candidate for splitting into 2 args `count($format, $options)`.